backend/models/grades.model.js

```diff
@@ -0,0 +1,60 @@
+const mongoose = require('mongoose');
+
+const gradeSchema = new mongoose.Schema(
+    {
+        sem1: {
+            type: String
+        },
+        sem1_pdf: {
+            type: String
+        },
+        sem2: {
+            type: String
+        },
+        sem2_pdf: {
+            type: String
+        },
+        sem3: {
+            type: String
+        },
+        sem3_pdf: {
+            type: String
+        },
+        sem4: {
+            type: String
+        },
+        sem4_pdf: {
+            type: String
+        },
+        sem5: {
+            type: String
+        },
+        sem5_pdf: {
+            type: String
+        },
+        sem6: {
+            type: String
+        },
+        sem6_pdf: {
+            type: String
+        },
+        sem7: {
+            type: String
+        },
+        sem7_pdf: {
+            type: String
+        },
+        sem8: {
+            type: String
+        },
+        sem8_pdf: {
+            type: String
+        }
```

Comment:
Also, why store the grades like this?

I would suggest making a `grade` array and then inside that array storing the objects of grade which include
- semester
- grade/pointer
- subject (for future if needed to store the details of each subject in the semester)